docs/src/main/asciidoc/logging.adoc

```diff
@@ -374,20 +376,21 @@ Add this extension to your build file as the following snippet illustrates:
     <artifactId>quarkus-logging-json</artifactId>
 </dependency>
 ----
-
++
 [source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
 .build.gradle
 ----
 implementation("io.quarkus:quarkus-logging-json")
 ----
-
++
 By default, the presence of this extension replaces the output format configuration from the console configuration, and the format string and the color settings (if any) are ignored.
 The other console configuration items, including those controlling asynchronous logging and the log level, will continue to be applied.
-
++
 For some, it will make sense to use humanly readable (unstructured) logging in dev mode and JSON logging (structured) in production mode.
 This can be achieved using different profiles, as shown in the following configuration.
-
-.Disable JSON logging in application.properties for dev and test mode
++
```

Comment:
This `+` looks weird to me, but I'm no expert.

Reply:
This is the problem of the level 5 heading. Michal mentioned that since that heading is so small, the space around that heading virtually doesnt exist, this that heading is shoulder to shoulder with that command box above.
I used pluses to glue that content together and make them alligned, thus it also creates some "additional" space for that tiny _Configuration_ heading bellow. I also remake these sentences into steps.
![image](https://github.com/quarkusio/quarkus/assets/48474054/d42e93a0-1599-4926-ab84-3cc91d871b6e)

Reply:
Works fine, looks good (when rendered) :)

Reply:
I don't have a strong opinion here but let's be cautious with this sort of tweaks: there's a good chance they might get broken over time because we don't usually do this.

So I would recommend to only use it when absolutely needed.